src/command/run.go

```diff
@@ -11,9 +11,9 @@ import (
 // Essential subshell commands are essential for the functioning of Git Town.
 // If they fail, Git Town ends right there.
 func MustRun(cmd string, args ...string) *Result {
-	result := RunInDir("", cmd, args...)
-	if result.Err() != nil {
-		fmt.Printf("\n\nError running '%s %s': %s", cmd, strings.Join(args, " "), result.Err())
+	result, err := RunInDir("", cmd, args...)
```

Comment:
To dry this up a bit should this use MustRunInDir instead of RunInDir?

Reply:
Good point, that's what I do in the next PR. This PR focuses on introducing the dedicated error.